apps/daimo-mobile/src/view/TabNav.tsx

```diff
@@ -73,6 +71,105 @@ export function TabNav() {
   );
 }
 
+export function TabNav() {
+  // Track whether we've onboarded. If not, show OnboardingScreen.
+  const [account] = useAccount();
+  const [isOnboarded, setIsOnboarded] = useState<boolean>(account != null);
+  useEffect(() => {
+    // This is a latch: if we clear the account, go back to onboarding.
+    if (isOnboarded && account == null) setIsOnboarded(false);
+  }, [isOnboarded, account]);
+  // Stay onboarding till it's complete.
+  const onOnboardingComplete = () => setIsOnboarded(true);
+
+  if (!isOnboarded) return <OnboardingScreen {...{ onOnboardingComplete }} />;
+
+  function forModalPresentationIOS({
```

Comment:
recommend adding a short comment explaining what this does. in particular, what's `current`, `next`, and `inverted`?

Reply:
Added 👌